packages/jest-circus/src/run.ts

```diff
@@ -149,6 +149,7 @@ const _callCircusHook = async ({
   testContext?: Circus.TestContext;
 }): Promise<void> => {
   await dispatch({hook, name: 'hook_start'});
+  hook.seenDone = false;
```

Comment:
do these in the `'hook_start'` and `'test_fn_start'` hooks rather than these utils directly